### ISSUE TITLE
[ExtensionListenerError] Extension 'emilast.LogFileHighlighter' FAILED to handle event: TypeError: Cannot read properties of undefined (reading 'visibleRanges')

### ISSUE THREAD

**Issue by ThaDaVos** [emilast/vscode-logfile-highlighter#613]
I am getting my `extension host` log spammed with these messages:
```
[error] Error: [ExtensionListenerError] Extension 'emilast.LogFileHighlighter' FAILED to handle event: TypeError: Cannot read properties of undefined (reading 'visibleRanges')
    at i.value (@APPDATA@\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:164:21030)
    at n.B (@APPDATA@\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:83:737)
    at n.C (@APPDATA@\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:83:807)
    at n.fire (@APPDATA@\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:83:1023)
    at c.$acceptEditorPropertiesChanged (@APPDATA@\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:156:34697)
    at E.S (@APPDATA@\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:151:5980)
    at E.Q (@APPDATA@\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:151:5746)
    at E.M (@APPDATA@\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:151:4778)
    at E.L (@APPDATA@\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:151:3605)
    at i.value (@APPDATA@\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:151:2297)
    at n.B (@APPDATA@\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:83:737)
    at n.fire (@APPDATA@\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:83:954)
    at s.fire (@APPDATA@\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:97:14453)
    at i.value (@APPDATA@\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:177:8655)
    at n.B (@APPDATA@\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:83:737)
    at n.fire (@APPDATA@\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:83:954)
    at s.fire (@APPDATA@\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:97:14453)
    at MessagePortMain.<anonymous> (@APPDATA@\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:177:6781)
    at MessagePortMain.emit (node:events:519:28)
    at Object.MessagePortMain._internalPort.emit (node:electron/js2c/utility_init:2:2285)
```

**Comment by emilast** [comment]
Thank you for reporting this, it will be investigated. Do you see any other problems than these log messages?

**Comment by ThaDaVos** [comment]
I haven't checked any further as I currently don't have logs available to highlight - but I can check

**Comment by ThaDaVos** [comment]
Just checked with the example log from the repo and it shows as follows when marked as "Log":
![afbeelding](https://github.com/user-attachments/assets/817efd57-9286-46b6-a7b2-fe3fca6e2be3)
When marked as "Plain Text":
![afbeelding](https://github.com/user-attachments/assets/dad9ff18-3257-433d-93dc-d5d721517742)

So it seems the extension does nothing, but I may be wrong

**Comment by ThaDaVos** [comment]
Some info - it does this when switching files it seems - and when no file opened - it spams the log

**Comment by emilast** [comment]
Thanks, I'll have a look at this during the weekend.

**Comment by emilast** [comment]
I just released a bug fix for this and it should work in 3.3.1. Please reopen this bug if it doesn't as I've had problems repeating the error consistently.

**Comment by emilast** [comment]
Correction: 3.3.2 should contain the correct fix. HTH
